src/components/Layout/MenuDrawer.jsx

```diff
@@ -26,36 +26,72 @@ const MenuDrawer = ({ open, setOpen }) => {
     Navigate("/attendance")
   }
 
+  const handleGoToMypage = () => {
+    Navigate("/mypage")
+  }
```

Comment:
```suggestion
  const handleGoToMypage = () => {
    Navigate("/mypage")
  }
```
이런 비슷한 코드가 너무 증가하면 파라미터 받는 형태로 변경할수도 있음!
이건 필수 사항은 아님!
```suggestion
  const handleNavigate = (destination) => {
    Navigate(`/${destination}`)
  }
```